osgp/shared/osgp-ws-publiclighting/src/main/resources/schemas/pl-adhocmanagement.xsd

```diff
@@ -287,6 +281,17 @@
     </xsd:sequence>
   </xsd:complexType>
 
+  <xsd:complexType name="LightSensorStatus">
+    <xsd:annotation>
+      <xsd:documentation>
+        Light sensor status represents the retrieved status of a light measurement device.
+      </xsd:documentation>
+    </xsd:annotation>
+    <xsd:sequence>
+      <xsd:element name="Dark" type="xsd:boolean" maxOccurs="1" />
```

Comment:
A specific type for `LightSensorStatusValue` (or another name) that has enumerated values for the sensor status would look a bit nicer I think. Then it would be immediately clear that the possible values are `Dark` or `Light` (I would imagine) and the meaning would not have to be derived from a boolean.